src/org/omegat/core/data/ProjectTMX.java

```diff
@@ -296,6 +296,11 @@ public void setTranslation(SourceTextEntry ste, TMXEntry te, boolean isDefault)
         }
     }
 
+    public void clear() {
+        defaults.clear();
+        alternatives.clear();
+    }
```

Comment:
```suggestion
    public synchronized void clear() {
        defaults.clear();
        alternatives.clear();
        // checkOrphanedCallback.clear();
    }
```

This should be synchronized and reset all internal states